### PR TITLE
Remove tsType in oas2tson output

### PR DESCRIPTION
**Pull request by jmjf**
Closes https://github.com/nearform/openapi-transformer-toolkit/issues/324

- Add DateExample to openapi.yml to demonstrate the behavior.
- Add tests for DateExample that pass with current output.
- Remove tsType in oas2tson (and red-green test).
- Update examples

I saw that `example/output/tson` duplicates `example/output\ts`. Knowing the maintainers prefer `ts`, I removed `tson`.

### CODE REVIEW

File: test/oas2tson.test.ts [hunk 1]
```diff
@@ -133,6 +134,62 @@ tap.test('oas2tson', async t => {
 `,
       'Customer.ts is created correctly'
     )
+
+    const OrderFile = resolveFromPackageRoot(outputPath, 'Order.ts')
+    const generatedOrderFile = fs.readFileSync(OrderFile, 'utf-8')
```
Comment (teobmg) [inline test/oas2tson.test.ts:139]:
```suggestion
    const orderFile = resolveFromPackageRoot(outputPath, 'Order.ts')
    const generatedOrderFile = fs.readFileSync(orderFile, 'utf-8')
```

File: test/oas2tson.test.ts [hunk 2]
```diff
@@ -133,6 +134,62 @@ tap.test('oas2tson', async t => {
...
+  },
+  title: "Order",
+  $id: "Order.json",
+} as const;
+`,
+      'Order.ts is created correctly'
+    )
+
+    const DateExampleFile = resolveFromPackageRoot(outputPath, 'DateExample.ts')
+    const generatedDateExampleFile = fs.readFileSync(DateExampleFile, 'utf-8')
```
Comment (teobmg) [inline test/oas2tson.test.ts:180]:
```suggestion
    const dateExampleFile = resolveFromPackageRoot(outputPath, 'DateExample.ts')
    const generatedDateExampleFile = fs.readFileSync(dateExampleFile, 'utf-8')
```

File: test/oas2tson.test.ts (hunk 1 above)
Comment (ilteoood) [inline test/oas2tson.test.ts:139]:
```suggestion
    const orderFile = resolveFromPackageRoot(outputPath, 'Order.ts')
    const generatedOrderFile = fs.readFileSync(orderFile, 'utf-8')
```

File: test/oas2tson.test.ts (hunk 2 above)
Comment (ilteoood) [inline test/oas2tson.test.ts:180]:
```suggestion
    const dateExampleFile = resolveFromPackageRoot(outputPath, 'DateExample.ts')
    const generatedDateExampleFile = fs.readFileSync(dateExampleFile, 'utf-8')
```

**Comment by jmjf** [comment]
Will do. I was following the pattern in other tests for oas2tson. Would you
like me to change those names too?

**Comment by jmjf** [comment]
Added two commits. One fixes the names identified in comments. The second fixes other names that used the same undesired pattern.

**Review by ilteoood — APPROVED** [review]

**Review by simoneb — APPROVED** [review]

**Pull request merged**